app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,330 @@
+
+
+}
+
+ $font-stack: 'Mako', sans-serif;
+
+ //colors
+ // $primary-color :
+ // $secondary-color
+ // $primary-brand-color
+ // $secondary-brand-color
+
+ h1, h2, h3, h4, h5 {
+   font-family: 'Lora', serif;
+   margin-bottom: 0rem;
+   margin-top: 0.5rem;
+ }
+
+ $primary-background-color: $pastel-orange; //  pastel-orange
+ $secondary-background-color: $light-green; // light-green
+ // in overrides
+ $primary-color: #ffffff; //  pastel-red
+ $secondary-color: $navy-blue; //dark-green
+
+ // Styles
+ body {
+   background-color: $dark-grey;
+   font-family: $font-stack;
+   font-weight: normal;
+ }
+
+ a {
+   color: $mustard-yellow;
+ }
+
+ // img {
+ //     display: inline-block
+ // }
+ //
+ a:hover, a:focus {
+   color: $mustard-yellow;
+ }
+
+ .flash-alert {
+   margin-top: 10px;
+ }
+
+ .review-form-list {
+   margin: 0;
+ }
+
+ .forgot-pass {
+   color: $light-green;
+ }
+
+ .forgot-pass a {
+   color: $light-green;
+ }
+
+ .review-title {
+   margin-bottom: 1rem;
+ }
+
+ .park-index-list-item {
+   background-color: $light-green;
+   list-style-type: none;
+   margin-bottom: 5px;
+   padding: 10px 15px;
+ }
+
+ .index-details {
+   color: $steel;
+   float: left;
+   font-size: 100%;
+   line-height: 2;
+ }
+
+ .index-rating p {
+   color: $steel;
+   float: center;
+   font-size: 300%;
+   line-height: 0.6;
+   position: relative;
+   top: 50%;
+   transform: translateY(-50%);
+ }
+
+ .park-info p {
+   position: relative;
+   top: 50%;
+   transform: translateY(-50%);
+   border-right: 2px solid;
+   border-color: $white;
+ }
+
+ .review-info p {
+   border-color: $white;
+   border-right: 2px solid;
+   position: relative;
+   top: 50%;
+   transform: translateY(-50%);
+ }
+
+ .park-icons img {
+   margin-right: 10px;
+   opacity: 0.6;
+   position: relative;
+   top: 50%;
+   transform: translateY(-50%);
+ }
+
+ .review-body p {
+   position: relative;
+   top: 50%;
+   transform: translateY(-50%);
+ }
+
+ .park-show-location {
+   margin-top: 0rem;
+ }
+
+ .park-show-rating {
+   margin-bottom: 1rem;
+ }
+
+ .show-icons {
+   margin-bottom: 10px;
+ }
+
+ .top-bar {
+   height: 3rem;
+ }
+
+ .new {
+   color: $white;
+ }
+
+ .no-bullet {
+   margin-bottom: 0;
+   margin-left: 0;
+ }
+
+ .button.small {
+   font-size: 1.2rem;
+   padding: 0.4rem;
+ }
+
+ .button.tiny {
+   font-size: 0.8rem;
+   padding: 0.4rem 0.6rem;
+ }
+
+ .button.tiny.vote {
+   margin: 0;
+ }
+
+ .button {
+   background-color: $pastel-red;
+ }
+
+ .button.alert {
+   background-color: $pastel-orange;
+ }
+
+ .has-form {
+
+   position: absolute;
+   top: 0;
+   left: 8rem; // adjust distance from left here
+   min-width: 14rem; // adjust width here
+
+   @media only screen and (max-width: 40em) {
+     min-width: 10rem;
+     top: 0.5rem;
+
+     .button {
+       height: 1.85rem;
+     }
+   }
+ }
+
+
+
+ .leaf {
+   background: url('http://static.pexels.com/wp-content/uploads/2014/07/beach-coast-holiday-2190-825x550.jpg') no-repeat center center;
```

Comment:
Prefer double-quoted strings<br>URL `http://static.pexels.com/wp-content/uploads/2014/07/beach-coast-holiday-2190-825x550.jpg` should not contain protocol or domain